### ISSUE TITLE
Handle malformed coordinates

### ISSUE THREAD

**Issue by SammysHP** [cgeo/cgeo#4887]
```
cgeo.geocaching.location.Geopoint$ParseException: N 55° 02,550 W 349° 49,975
at cgeo.geocaching.location.GeopointParser.parse(GeopointParser.java:73)
at cgeo.geocaching.location.Geopoint.<init>(Geopoint.java:57)
at cgeo.geocaching.NavigateAnyPointActivity$CoordDialogListener.onClick(NavigateAnyPointActivity.java:288)
```

**Comment by samueltardieu** [comment]
What should be cgeo behaviour in this case?

**Comment by SammysHP** [comment]
Show a warning that it was not able to understand the coordinates?

**Comment by rahu** [comment]
I assumed this would be easy to fix and a perfect bug for me to come back into cgeo coding. But I'm unable to reproduce this? 
Main screen -> go to -> click on latitude -> at cgeo.geocaching.NavigateAnyPointActivity$CoordDialogListener.onClick is called but with the old coords. If I enter invalid coords in the next screen I get a warning..

**Comment by Lineflyer** [comment]
@rahu Did you try all of the offered formattings (I suspect it could happen in "plain" mode)

**Comment by rahu** [comment]
I tried all formats now. I think this has been fixed by  @samueltardieu with this commit 918732d2d010fc417f4305084e3924e7060e295d -> issue #3755

**Comment by SammysHP** [comment]
Interesting. It handles the case from above, but this issue is newer than #3755. But I'm not sure if I noticed it myself or got it from someone else. Anyway, closing.
